src/Interpreters/AggregateDescription.h

```diff
@@ -21,4 +26,52 @@ struct AggregateDescription
 
 using AggregateDescriptions = std::vector<AggregateDescription>;
 
+
+struct WindowFunctionDescription
+{
+    // According to the standard, a window function can refer to a window declared
+    // elsewhere, using its name. We haven't implemented this yet: all windows
+    // are declared in OVER clause, but they still get an auto-generated name,
+    // and this name is used to find the window that corresponds to a function.
+    std::string window_name;
```

Comment:
If we sore functions into a window, do we have to store window name again?
Should `window_name` for each `WindowFunctionDescription` be the same as `window_name` from `WindowDescription`?